src/lib/preferences-default-codes.ts

```diff
@@ -29,6 +29,9 @@ const ywLogo = `
 // ---------------------------------------------------------------------------------------------
 
 export const defaultPreferencesContent = `
+    const { osWidgets, rxjs} = await webpmClient.install({
+        modules:['@youwol/os-widgets#^0.2.1 as osWidgets' , 'rxjs#^7.5.6 as rxjs' ],
```

Comment:
just a matter of formatting: can you put each elements of 'modules' in a new line (with appropriate indentation)